app/templates/_gulpfile.js

```diff
@@ -307,6 +307,8 @@ gulp.task('npm-package', (cb) => {
   pump(
     [
       gulp.src(['README.md', 'LICENSE', 'CHANGELOG.md',
```

Comment:
This is only needed when targeting **angular 2.x.x**, so please update as follow:
```
gulp.src(['README.md', 'LICENSE', 'CHANGELOG.md',<% if(ngVersion === '2.0.0'){ %>
       `${config.buildDir}/lib-es5/**/*.js`,
       `${config.buildDir}/lib-es5/**/*.js.map`,<% } %>
```